File: app/front/src/components/main/dto/add_xml/updateWkTblAddByXmlCapsuleDto.ts
import { WkTblMasterAllByXmlEntity, type WkTblMasterAllByXmlEntityInterface } from "../../entity/wkTblMasterAllByXmlEntity";
import { type FrameworkCapsuleDtoInterface ,FrameworkCapsuleDto } from "seijishikin-jp-normalize_common-tool";

interface UpdateWkTblAddByXmlCapsuleDtoInterface extends FrameworkCapsuleDtoInterface {

    /** 編集対象Entity */
    wkTblMasterAllByXmlEntity: WkTblMasterAllByXmlEntityInterface;

}

/**
 * 企業／団体一括登録履歴ワークテーブル更新CapsuleDto
 */
class UpdateWkTblAddByXmlCapsuleDto extends FrameworkCapsuleDto
    implements UpdateWkTblAddByXmlCapsuleDtoInterface {

    /** 編集対象Entity */
    wkTblMasterAllByXmlEntity: WkTblMasterAllByXmlEntityInterface;

    constructor() {
        super();
        this.wkTblMasterAllByXmlEntity = new WkTblMasterAllByXmlEntity()
    }
}


export { type UpdateWkTblAddByXmlCapsuleDtoInterface, UpdateWkTblAddByXmlCapsuleDto }

File: app/back/src/main/java/net/seijishikin/jp/normalize/manage/kanrensha/dto/add_xml/UpdateWkTblAddByXmlCapsuleDto.java
package net.seijishikin.jp.normalize.manage.kanrensha.dto.add_xml;

import java.io.Serializable;

import net.seijishikin.jp.normalize.common_tool.dto.FrameworkCapsuleDto;
import net.seijishikin.jp.normalize.manage.kanrensha.entity.WkTblMasterAllByXmlEntity;


/**
 * XML一括登録履歴ワークテーブル更新CapsuleDto
 */
public class UpdateWkTblAddByXmlCapsuleDto extends FrameworkCapsuleDto // NOPMD DataClass
        implements Serializable {

    /** Serialize id */
    private static final long serialVersionUID = 1L;

    /** 編集対象Entity */
    private WkTblMasterAllByXmlEntity wkTblMasterAllByXmlEntity = new WkTblMasterAllByXmlEntity();

    /**
     * 編集対象Entityを取得する
     *
     * @return 編集対象Entity
     */
    public WkTblMasterAllByXmlEntity getWkTblMasterAllByXmlEntity() {
        return wkTblMasterAllByXmlEntity;
    }

    /**
     * 編集対象Entityを設定する
     *
     * @param wkTblMasterAllByXmlEntity 編集対象Entity
     */
    public void setWkTblMasterAllByXmlEntity(final WkTblMasterAllByXmlEntity wkTblMasterAllByXmlEntity) {
        this.wkTblMasterAllByXmlEntity = wkTblMasterAllByXmlEntity;
    }

}
